### PR TITLE
Add sidebar menu with larivaar assist option

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
https://github.com/KhalisFoundation/learn-larivaar/assets/1990932/88c91d96-995d-4dc5-986c-a3b228735b8f

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -1,73 +1,27 @@
...
   return (
-    <SafeAreaView>
+    <LarivaarContext.Provider value={value}>
       <NavigationContainer>
-        <Switch
-          value={larivaarAssist}
-          onValueChange={() => setLarivaarAssist(!larivaarAssist)}
-        />
+        <Drawer.Navigator
+          // eslint-disable-next-line react/no-unstable-nested-components
```
Comment (theamanjs) [inline App.tsx:19]: Should change the code according to this https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unstable-nested-components.md

File: components/Launchpad/lauchpad.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import {layoutStyles, elementStyles} from '../../styles';
+
+const Launchpad = (): JSX.Element => {
+  const isDarkMode = useColorScheme() === 'dark';
+  const textInputRef = useRef<TextInput>(null);
+
+  const [inputAng, setInputAng] = useState(1);
+  return (
+    <SafeAreaView>
+      <StatusBar barStyle={isDarkMode ? 'light-content' : 'dark-content'} />
```
Comment (theamanjs) [inline components/Launchpad/lauchpad.tsx:23]: the condition is logically opposite

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: App.tsx (hunk 1 above)
Comment (Gauravjeetsingh) [reply App.tsx:19]: @theamanjs veerji, this is fixed in pull request #32

File: components/Launchpad/lauchpad.tsx (hunk 2 above)
Comment (Gauravjeetsingh) [reply components/Launchpad/lauchpad.tsx:23]: It appears that way, but the light-content means white text in status bar, and the dark-content means black colored text in status bar. 

So in dark mode, we need light content (white colored text) in status bar else we need dark content (black colored text)

Comment (theamanjs) [reply components/Launchpad/lauchpad.tsx:23]: Umm okay

**Review by theamanjs — APPROVED** [review]

**Pull request merged**